### DISCUSSION TITLE
user can't create a project

### DISCUSSION THREAD

**Discussion by merlinbest5** [biigle/core#961] [Q&A]
one of my colleagues is unable to create their own project, and it's unclear to me how to resolve this. see the error message attached.
![image (5)](https://github.com/user-attachments/assets/399a1316-e108-4086-92e5-aa88222398da)

**Comment by dlangenk** [comment]
Hi Merlin,

before granting full rights to a user, we check that there is no previous account and that the user is a real user and not spam or a bot. This is done manually and may take some time. Is this on a private biigle instance or at biigle.de. If this is a private biigle run by your institute the admin will get an email and can grant access to that user. Currently in our instance (biigle.de) I don't see any open tickets for granting full user rights. If this is however at biigle.de please write an email to info @ biigle.de and I'll have a look.

**Reply by merlinbest5** [reply]
it's on the public biigle but they did mention having 2 accounts - one made with our government email after working as an external contractor initially. i'll send an email shortly, thanks daniel.

**Reply by dlangenk** [reply]
Ah ok. There might me some ongoing email conversation then, I didn't have a look at yet. Unfortunately, we are a little bit shorthanded currently, because of illness.
